CONTRIBUTING.md

```diff
@@ -58,11 +58,15 @@ request](https://help.github.com/articles/creating-a-pull-request/).
 
 To setup the repository:
 
-1. `bundle install`
+1. `BUNDLE_GEMFILE=gemfiles/rails-7.1.gemfile bundle install`
```

Comment:
This will get stale. What's wrong with main? I know sometimes it fails but that's ok - it gives us a heads up on what is changing sometimes.